crates/derive/src/stages/channel_bank.rs

```diff
@@ -88,7 +88,9 @@ where
         });
 
         // Check if the channel is not timed out. If it has, ignore the frame.
-        if current_channel.open_block_number() + self.cfg.channel_timeout < origin.number {
+        if current_channel.open_block_number() + self.cfg.channel_timeout(origin.timestamp) <
```

Comment:
Oh wait actually - I don't think we should be using the origin timestamp here, yeah? We want an L2 timestamp for the L2 fork activation.

Reply:
Oh shoot you're right, good catch

Reply:
nvm https://github.com/ethereum-optimism/optimism/blob/develop/op-node/rollup/derive/channel_bank.go#L101